Dockerfile

```diff
@@ -93,19 +93,63 @@ RUN pip install -e . --no-deps
 ###################
 ## Frontend Base ##
 ###################
-FROM node:16-slim as frontend
+FROM node:16-alpine as frontend
+
+RUN apk add --no-cache libc6-compat
+# Build the frontend clients
+WORKDIR /fides/clients
+COPY clients/package.json ./
+COPY clients/package-lock.json ./
+COPY clients/fides-consent/package.json ./fides-consent/package.json
+COPY clients/admin-ui/package.json ./admin-ui/package.json
+COPY clients/privacy-center/package.json ./privacy-center/package.json
 
-# Build the admin-ui frontend
-WORKDIR /fides/clients/admin-ui
-COPY clients/admin-ui/package.json clients/admin-ui/package-lock.json ./
 RUN npm install
-COPY clients/admin-ui/ .
+
+COPY clients/ .
 
 ####################
 ## Built frontend ##
 ####################
 FROM frontend as built_frontend
-RUN npm run export
+
+# Builds and exports admin-ui
+RUN npm run export-ui
+# Builds privacy-center
+RUN npm run build-pc
+
+#############################
```

Comment:
The nittiest of all nits:

The `#` should match the length of the title line

Reply:
hahaha will fix thanks!